modules/installation-approve-csrs.adoc

```diff
@@ -107,7 +107,7 @@ endif::ibm-z,ibm-z-kvm[]
 +
 [NOTE]
 ====
-Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. After you approve the initial CSRs, the subsequent node client CSRs are automatically approved by the cluster `kube-controller-manager`.
+Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. Once the client CSR is approved, the Kubelet creates a secondary CSR for the serving certificate, which requires manual approval. Then subsequent serving certificate renewal requests are automatically approved by the cluster-machine-approver if the Kubelet requests a new certificate with identical parameters.
```

Comment:
Then,

Reply:
Yes, it should just be `machine-approver` in back ticks. Good catch ty!